docs/readme.md

```diff
@@ -4,29 +4,31 @@ Data API builder for Azure Databases provides modern REST and GraphQL endpoints
 
 With Data API builder, database objects can be exposed via REST or GraphQL endpoints so that your data can be accessed using modern techniques on any platform, any language, and any device. With an integrated and flexible policy engine, native support for common behavior like pagination, filtering, projection and sorting, the creation of CRUD backend services can be done in minutes instead of hours or days, giving developers an efficiency boost like never seen before.
 
-## Table of content
+## Table of contents
```

Comment:
Dont need this table of contents. I meant to provide the instructions on how to do external contributor doc edits in the README.md file that serves as the "placeholder" NOT literally named as placeholder.

Reply:
Oops I misunderstood this earlier, I'll update this